.travis.yml

```diff
@@ -26,8 +26,6 @@ script:
   - ./scripts/upload-artifacts.sh
 
 jobs:
-  # allow_failures:
-  #   - name: "🍳 Testing"
```

Comment:
not sure, but in the last PR only a stage is built, I guess there is an issue here
https://travis-ci.org/opengisch/QField/builds/650273752?utm_source=github_status&utm_medium=notification